.github/workflows/execution-performance.yaml

```diff
@@ -8,15 +8,15 @@ on:
 
 jobs:
   execution-performance:
-    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-execution-performance.yaml@main
+    uses: ./.github/workflows/workflow-run-execution-performance.yaml #aptos-labs/aptos-core/.github/workflows/workflow-run-execution-performance.yaml@main
```

Comment:
I'll also roll these types of changes back on landing. They're just to make sure I'm testing the actual actions (that aren't on the main branch yet 😄)